docs/content/documentation/getting-started/installation.md

```diff
@@ -49,6 +49,7 @@ To build it from source, you will need to have Git, [Rust (at least 1.30) and Ca
 installed. You will also need additional dependencies to compile [libsass](https://github.com/sass/libsass):
 
 - OSX, Linux and other Unix: `make` (`gmake` on BSDs), `g++`, `libssl-dev`
+	- NixOS: Invoke `nix-shell` in the cloned directory. This opens a shell with the above dependencies. You then run the below command to build the release.
```

Comment:
looks like it has a space before the `-`?

Reply:
@Keats indeed, I wanted it to be a subpoint of the above instruction